sdk/nodejs/resource.ts

```diff
@@ -407,7 +407,7 @@ export class Output<T> {
         };
 
         this.get = () => {
-            throw new RunError(`Cannot call '.get' during update or preview.
+            throw new Error(`Cannot call '.get' during update or preview.
```

Comment:
this is a general error during normla control flow.  It shouldn't really be associated with any specific resource.  Users will get the callstack though now, which will tell them where the problem is.